acc_x_bias: 0.166886131633737
acc_y_bias: 0.000429187930536
acc_z_bias: 0.002035899057871
gyro_x_bias: -0.000321044850675
gyro_y_bias: 0.000389816547892
gyro_z_bias: 0.001530990329192
roll_offset: -0.000241650835166
pitch_offset: -0.016646670210113
